Extensions/DialogueTree/dialoguetools.js

```diff
@@ -505,6 +523,7 @@ gdjs.dialogueTree.goToNextDialogueLine = function() {
       time: this.dialogueText.length + offsetTime,
     });
     this.dialogueData = this.dialogue.next().value;
+    this.dialogueDataType = 'command';
```

Comment:
What's the reason for moving this?

Reply:
I tried to see if it fixes the bug I described, but it doesnt really do it. I guess I can undo this